EduID/Flows/PersonalInfo/PersonalInfoViewModel.swift

```diff
@@ -13,8 +13,13 @@ class PersonalInfoViewModel: NSObject {
     
     var viewController: CreateEduIDAddInstitutionViewController?
     
-    override init() {
+    init(_ loadData: Bool) {
         super.init()
+        if loadData {
+            Task {
+                await getData()
+            }
```

Comment:
getData isn't async.

Reply:
```
    @MainActor
    func getData() {
        Task {
            do {
                try await userResponse = UserControllerAPI.meWithRequestBuilder()
                    .execute()
                    .body
                processUserData()
            } catch {
                dataFetchErrorClosure?(error)
            }
        }
    }
    
    private func processUserData() {
```

`func getData` shouldn't have `@MainActor` but `func processUserData` should.

Reply:
well, that's what I did first build de Xcode Editor complained!
`Call to main actor-isolated instance method 'getData()' in a synchronous nonisolated context`
and
`Expression is 'async' but is not marked with 'await'`